Dockerfile

```diff
@@ -7,10 +7,9 @@ ENV JENKINS_MODE exclusive
 
 ENV SWARM_PARAMS "'-labels slave -executors 1'"
 
-# To avoid error: sudo: sorry, you must have a tty to run sudo
-RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
-
-#RUN usermod -u $UID omero
+# Change user id to fix permissions issues
+ARG USER_ID=1000
```

Comment:
This is a nice idea. I almost wonder if it should have been in the omero-ssh-c7 itself, but that would be harder to use for devspace so :+1: